test/Microsoft.Health.Dicom.Tests.Integration/Microsoft.Health.Dicom.Tests.Integration.csproj

```diff
@@ -27,6 +27,7 @@
     <PackageReference Include="Polly" Version="7.2.1" />
     <PackageReference Include="xunit" Version="2.4.1" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3" />
+    <PackageReference Include="Microsoft.SqlServer.DACFx" Version="150.4897.1" />
```

Comment:
Is this required?

Reply:
yes, it's required for database schema comparison https://github.com/microsoft/dicom-server/blob/feature/customtags/test/Microsoft.Health.Dicom.Tests.Integration/Persistence/SqlServerSchemaUpgradeTests.cs

Reply:
it's copied from FHIR https://github.com/microsoft/fhir-server/blob/8fb192f6a9fc4fad735182e683340e6b5900e92a/test/Microsoft.Health.Fhir.R5.Tests.Integration/Microsoft.Health.Fhir.R5.Tests.Integration.csproj#L10